Source/Blazorise/Base/BaseContext.cs

```diff
@@ -0,0 +1,26 @@
+﻿#region Using directives
+#endregion
+
+namespace Blazorise
+{
+    /// <summary>
+    /// Holds the Context for an Item based component.
+    /// </summary>
+    /// <typeparam name="TItem">Type of an item.</typeparam>
+    public abstract class BaseContext<TItem>
```

Comment:
`BaseContext` seems too generic of a name. It should be obvious to know the Context of what it represents?

Reply:
`BaseTemplateContext`?

Reply:
`BaseRenderFragmentContext` 
`BaseFragmentContext`